### DISCUSSION TITLE
Saving window size and position

### DISCUSSION THREAD

**Discussion by Nadias-jp** [mpdifran/Memento-Issues#38] [Ideas]
When I change the size and position of a window on my Mac, it is not saved.
It would be useful to save them.

**Comment by mpdifran** [comment]
Thanks for the suggestion! This definitely sounds do-able, I'll make sure to schedule this.

**Reply by Nadias-jp** [reply]
Thank you.
I'm looking forward to it.

**Comment by mpdifran** [comment]
Logged this task to track this: https://github.com/mpdifran/Memento-Issues/issues/41
